s30310-KacperKoszelak/zajecia3/3_3.txt

```diff
@@ -30,11 +31,13 @@ Zobacz co się stanie:
 W opisie rozwiązania zadania umieść:
 1. jak rozumiesz co się stało?
     -zawartość "wynik3_1.txt" nie została zapisana, a zawartość "wynik3_2.txt" 
-    została zachowana
+     została zachowana
 
 2.czym oba te przykłady się różnią?
-    -CTRL+C nie zapisuje wprowadzanego tekstu, a CTRL+D zapisuje
+    -CTRL+C natychmiastowo zabija obecnie wykonywany proces, a CTRL+D przetwarza 
+    wpisane dane i kończy pracę
+
 
 3.DLACZEGO się różnią?
-    -CTRL+C natychmiastowo przerywa wykonywanie programu, CTRL+D zwraca tekst z 
-    programu i go kończy
+    -Oba skróty mają inne zastosowania, CTRL+C służy do zabicia procesu, a 
+	CTRL+D zapisuje treść na swoim standardowym wyjściu

```

Comment:
^D nic nie zapisuje. To także nie ma żadnego wyjścia. Brak zrozumienia materiału.
Do poprawy.